src/App.js

```diff
@@ -27,15 +28,11 @@ class App extends React.Component {
           </a>
         </nav>
         <div className="container">
-          <p className="text-center mb-4">Browse all the recipes here, select a category or search for an ingredient.</p>
-          <div className="row">
-            {recipes.map((recipe, i) =>
-              <RecipeCard
-                recipe={recipe}
-                key={i}
-              />
-            )}
-          </div>
+          <Router>
+            <Route path="/">
+              <HomePage recipes={recipes}/>
```

Comment:
I think it’s best if we do the actual call to the api inside the component. In this case, when we visit another page, we don’t need to do that api call